scripts/local/dev-component-library-install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+GREEN='\033[1;32m'
+ENDCOLOR='\033[0m'
+
+read -p "Which branch? " BRANCH
+
+echo -e "${GREEN}Move into atomic and checkout develop"
+cd web/themes/contrib/atomic
+git checkout develop
```

Comment:
I ran into an issue with npm not using the correct version. I did an ```nvm use``` from the atomic directory and that fixed it. So, perhaps after line 9, add an ```nvm use```

Reply:
hmm... that's assuming people have nvm installed, but I think that's a standard development tool - at least at 4K. I'll add it for now, and we can tweak it later, if we need to.